### ISSUE TITLE
PRO PortalList not honered for "NAMED_USER" AuthorizationType

### ISSUE THREAD

**Issue by pfoppe** [Esri/arcgis-powershell-dsc#217]
We are attempting to push ArcGIS PRO to a machine, but the **PortalList** setting is not honored.  Our authorization type is NAMED_USER.  

Looking at the underlying code, it seems that there is an if statement preventing setting the PortalList for the NAMED_USER: 
https://github.com/Esri/arcgis-powershell-dsc/blob/037b14af2d058f54c96a0f46f4facd9baf77579f/Modules/ArcGIS/Configurations-OnPrem/ArcGISInstall.ps1#L389

Here is the JSON contents (redacted): 
```
{
    "AllNodes": [
        {
            "NodeName": "<MACHINE>",
            "Role": [
                "Pro"
            ]
        }
    ],
    "ConfigData": {
        "ProVersion": "2.4",
		"SevenZipInstallerPath": "\\\\<server>\\<share>\\InstallMedia\\7Zip\\7z938-x64.msi",		
        "Pro":{
            "AuthorizationType":"NAMED_USER",
            "BlockAddIns":"0",
            "SoftwareClass":"Professional",
            "AllUsers":"1",
            "EnableEUEI":"False",
            "CheckForUpdatesAtStartup":"False",
			"PortalList":"https://<MACHINE>.<DOMAIN>/portal",
            "Installer": {
                "Path": "\\\\<server>\\<share>\\InstallMedia\\Esri\\pro\\2.4\\ArcGISPro_24_169232.exe",
                "InstallDir": "C:\\Program Files\\ArcGIS\\Pro",
                "PatchesDir":"\\\\<server>\\<share>\\InstallMedia\\Esri\\pro\\patches"
            }
        }
    }
}
```

Thanks!

**Comment by cameronkroeker** [comment]
Thanks @pfoppe, We plan to fix this in the next release.

**Comment by pfoppe** [comment]
Thanks!

**Comment by cameronkroeker** [comment]
@pfoppe 

This has been addressed in the latest release, v3.0.0.
